subdl

```diff
@@ -168,7 +169,7 @@ def SearchSubtitlesByIMDBId(filename, langs_search, imdb_id):
     return data and [SubtitleSearchResult(d) for d in data]
 
 def SearchSubtitlesByFileName(filename, langs_search):
-    file_name, file_ext = os.path.splitext(os.path.basename(filename))
+    file_name = file_base(filename)
```

Comment:
This would cause file_name to include the complete path, without extension. In the previous code the base name is used to just get the filename part, without extension, of the path.